frontend/src/stimulus/controllers/dynamic/overview/project-life-cycles-form.controller.ts

```diff
@@ -31,22 +31,13 @@
 import { Controller } from '@hotwired/stimulus';
 
 export default class ProjectLifeCyclesFormController extends Controller {
```

Comment:
Have you thought about letting `ProjectLifeCyclesFormController` inherit from `FormPreviewController`? Thus you would not have register two controllers and handle the dispatched event in the html but can do it directly in the controller.

Reply:
My idea was that the `FormPreviewController` can also be used as a standalone controller and wanted to communicate with it that way.
However, inheritance in this case would definitely make things simpler. The dispatching logic could be spared.

Reply:
I changed the `ProjectLifeCyclesFormController` to inherit from the `FormPreviewController`.
I also cherry picked a bugfix related to this code from the release branch and updated the docs.